apps/bttester/src/btp_gap.c

```diff
@@ -496,7 +496,7 @@ start_advertising(const void *cmd, uint16_t cmd_len,
         i += ad[adv_len].data_len;
     }
 
-    for (sd_len = 0U; i < cp->scan_rsp_len; sd_len++) {
+    for (i = 0, sd_len = 0U; i < cp->scan_rsp_len; sd_len++) {
```

Comment:
this will cause scan response to set from adv data...  I think (not tested) that correct fix would be to change loop end condition here to check for both ad and sd len